templates/python-package-to-windows-webapp-on-azure.yml

```diff
@@ -0,0 +1,101 @@
+# Python package to Windows Web App Azure
+# Build your Python project and deploy it to Azure as a Windows Web App.
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+trigger:
+- master
+
+variables:
+  # Azure Resource Manager connection created during pipeline creation
+  azureServiceConnectionId: '{{ azureServiceConnection.Id }}'
+  
+  # Web app name
+  webAppName: '{{ webAppName }}'
+
+  # Agent VM image name
+  vmImageName: 'windows-latest'
+
+  # Environment name
+  environmentName: '{{ webAppName }}'
```

Comment:
Given that it's only used once, perhaps you can put the `{{ webAppName }}` directly?

Reply:
This is just some default value we are trying to populate for the user. We expect this to be edited by the user to suite their needs. Making it a variable will make it easy for them to find it and edit. Would not want to hide it deep in the script.

Reply:
That won't be the case right, we would have already created an environment with that name by then?